x/params/client/utils/utils.go

```diff
@@ -0,0 +1,25 @@
+package utils
+
+import (
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/x/params"
+)
+
+// TODO: remove hardcoded storename
+const storeName = "params"
+
+// Query parameters from node with CLIContext
```

Comment:
Should we instead implement a querier?

Reply:
I'd opt to do it another PR. Other things are blocking on this PR.